### PR TITLE
Update foreman-scc foreman-tasks requirement

### PR DESCRIPTION
**Pull request by zjhuntin**

### CODE REVIEW

File: packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec [hunk 1]
```diff
@@ -24,13 +24,13 @@ Requires: %{?scl_prefix_ruby}ruby(release)
 Requires: %{?scl_prefix_ruby}ruby
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix}rubygem(foreman-tasks) >= 0.10
-Requires: %{?scl_prefix}rubygem(foreman-tasks) < 1
+Requires: %{?scl_prefix}rubygem(foreman-tasks) < 2
```
Comment (tbrisker) [inline packages/katello/rubygem-foreman_scc_manager/rubygem-foreman_scc_manager.spec]: this doesn't match the gemspec - https://github.com/ATIX-AG/foreman_scc_manager/blob/master/foreman_scc_manager.gemspec#L25 
we should drop the upper limit

**Comment by tbrisker** [comment]
actually, we need a new release of scc_manager with the updated spec, @sbernhard - can you please make one?

**Comment by sbernhard** [comment]
Colleage is already working on it.

**Comment by tbrisker** [comment]
closing for #4636

**Pull request closed without merging**